opflexagent/config.py

```diff
@@ -57,6 +57,15 @@
     cfg.StrOpt('fabric_bridge', default='br-fabric',
                help=_("The name of the bridge which connects to the ACI "
                       "fabric")),
+    cfg.StrOpt('bridge_manager',
```

Comment:
Either name works for me -- I picked bridge_manager only because that's the base class that these inherit from. However, I think it makes sense to keep FakeManager in the same file as OvsManager, since it's just a specialization of the OvsManager class.

Reply:
Ok, please ignore my comments, I agree with you. I incorrectly assumed what fake meant here, .  I am assuming nova is just adding the ports to ovsdb directly in this case.  I see that the specific bridge manager class does not have to be imported in gbp_agent as well. I will use this method as well. Thanks!